internal/pkg/cli/deploy/static_site.go

```diff
@@ -205,18 +205,13 @@ func (d *staticSiteDeployer) convertSources() ([]manifest.FileUpload, error) {
 	return convertedFileUploads, nil
 }
 
-func (d *staticSiteDeployer) validateAlias() error {
-	if d.staticSiteMft.HTTP.Alias == "" {
+func (d *staticSiteDeployer) validateAppDomainAlias() error {
+	if d.staticSiteMft.HTTP.Alias == "" || d.staticSiteMft.HTTP.Certificate != "" {
```

Comment:
Is this a valid configuration or do we need to error out `alias must be specified when importing a certificate` just like Load Balanced Service.
```
http:
 certificate: "arn:aws:acm:us-east-1:197732814171:certificate/60323534-ffc3-4d20-97b2-56dd2754d0c7"
```

Reply:
That's a good question. I'll add to the manifest validation.